scripts/opensearch-entrypoint.sh

```diff
@@ -38,4 +38,8 @@ while [[ $e != "200" ]]; do
 done
 
 # run testing tool
-su test -c "python3 knn-perf-tool.py $OKPT_COMMAND $OKPT_CONFIG_PATH $OKPT_OUTPUT_PATH"
+if [[ -z $OKPT_LOG_LEVEL ]]; then
```

Comment:
Wondering if there is a way to avoid this if statement. Can you think of any?

Reply:
I've set `info` as the default log level instead.